src/domain/services/FileConfigService.ts

```diff
@@ -130,5 +135,7 @@ export class FileConfigService implements OnModuleInit, IFileModuleConfig {
             'image/webp',
             'image/svg+xml',
         ];
+
+        this.saveTemporaryFileAfterUpload = Boolean(process.env.SAVE_TEMPORARY_FILE_AFTER_UPLOAD);
```

Comment:
Предлагаю всё же отказаться от переменной окружения в модуле nest-file, и здесь использовать значение из `custom`, например так:
```suggestion
        this.saveTemporaryFileAfterUpload = Boolean(custom.saveTemporaryFileAfterUpload);
```